app/views/docs/rate-limits.phtml

```diff
@@ -79,3 +79,14 @@ Connection: close
 }
 </code></pre>
 </div>
+
+<h2><a href="/docs/rate-limits#disable-limits" id="disable-limits">Disable Rate Limits</a></h2>
+<p>During development, you may need to frequently make requests to certain endpoints to test your code. You can temporarily disable rate limiting to prevent triggering rate limits during development.</p>
+<p>You can disable rate limiting by setting <span class="tag">_APP_OPTIONS_ABUSE</span> to <span class="tag">disabled</span> in the <b>.env</b> file found in your Appwrite install directory.</p>
```

Comment:
I think this section can be shorter and just refer to the env variables doc, instead of trying to re-explain how to update them. We also miss information on recreating the Appwrite container in the last paragraph, which is maybe another hint to move this info to the relevant page.

We should also have a self-hosted only reminder when Cloud goes live.